### ISSUE TITLE
migrate.go no longer supports creating databases with dashes in the name

### ISSUE THREAD

**Issue by joshgarnett** [heroiclabs/nakama#776]
https://github.com/heroiclabs/nakama/pull/606 changed the behavior of the how Nakame creates databases from:

```
if _, err = db.Exec(fmt.Sprintf("CREATE DATABASE %q", dbname)); err != nil {
```

to

```
if _, err = db.Exec(fmt.Sprintf("CREATE DATABASE %s", dbname)); err != nil {
```

This means the migration code fails on create if you have a dash in the name. 
```
{"level":"info","ts":"2022-02-06T13:30:39.421Z","caller":"migrate/migrate.go:201","msg":"Creating new database","name":"some-dash-name"}
{"level":"fatal","ts":"2022-02-06T13:30:39.433Z","caller":"migrate/migrate.go:211","msg":"Failed to create database","error":"ERROR: syntax error at or near \"-\" (SQLSTATE 42601)"}
```

**Comment by novabyte** [comment]
Thanks @joshgarnett nice catch. Looks like a small regression. We'll resolve it as part of the next release.

**Comment by zyro** [comment]
Fixed in 9d2ff19da62df5f0bff7514103e53c4135ef19f0 and will be part of the next release.
